beeline/src/java/org/apache/hive/beeline/HiveSchemaTool.java

```diff
@@ -159,12 +159,12 @@ Connection getConnectionToMetastore(boolean printInfo)
 
   private NestedScriptParser getDbCommandParser(String dbType, String metaDbType) {
     return HiveSchemaHelper.getDbCommandParser(dbType, dbOpts, userName,
-	passWord, hiveConf, metaDbType);
+	passWord, hiveConf, metaDbType, false);
```

Comment:
Looks like most of the methods are passing the boolean argument as false. May be we keep it default to false instead of passing it as a input argument for all the method calls.